internal/model/sql/postgres/schema/hellper.sql

```diff
@@ -26,6 +26,7 @@ CREATE TABLE public.incident (
 	channel_id varchar(50) NULL,
   commander_id text NULL,
   commander_email text NULL,
+  type varchar(50) NULL,
```

Comment:
Why not use a Boolean type? I would change it to "is_silent", and if the modal is equal to NO then it doesn't change, but if it is equal to YES, it updates to true. By default, it could set to false.

Reply:
I even considered putting a Boolean column, but I chose to save literally the channel type in the database so that we could use that column for more features in the future. Maybe that intention would be more evident if I put "channel_type" instead of "type"

Reply:
It makes sense, but if you don't want to notify in some other type?

Reply:
The PR notifies only in public incidents, any other type of incident is not notified. It was done that way because to day the project has an architecture extremely linked to the incident process that we have today in the RD, however I think it is important to consider making the project more and more flexible, so an improvement of this PR would, for example, create an environment variable that keeps the incidents that the user wants to be notified.